### ISSUE TITLE
Unable to replace Mita's model

### ISSUE THREAD

**Issue by Anua831** [Rist8/MiSide-UniversalAssetLoader#10]
When I try to replace the mita model with the fbx file, an error occurs. I have double checked that the model is in the asset folder and the addons_config.txt is also confirmed.
![YS4F`KB(TERJIKQCU62K45T](https://github.com/user-attachments/assets/15680277-b90c-45e2-a42d-14e333bb0800)
![BWPT%OJIU`9YTM`HDZ{%3D3](https://github.com/user-attachments/assets/3f35d3ed-7185-4572-828a-5ef4687f54c9)
![C_B850Q@N`835 }(AM W{2](https://github.com/user-attachments/assets/cd68de1d-ab3a-4f03-a114-99c758f6d573)

**Comment by Anua831** [comment]
![BWPT%OJIU`9YTM`HDZ{%3D3](https://github.com/user-attachments/assets/0c64a8a8-4ee9-4bc6-8577-be59a2d6c9d6)
z

**Comment by Rist8** [comment]
Probably some problem with a position of your mesh in your .fbx file's hierarchy, compare your .fbx file with an example MitaCatMaid.fbx and see what s the difference between the SHair mesh and CatEars mesh and try to fix the difference

**Comment by Anua831** [comment]
I mimicked the grid position in the MitaCatMaid.fbx file hierarchy as best as possible, but it had no effect
![)S2CS06A98I{HR6XIU}3K F](https://github.com/user-attachments/assets/3c8189ec-891a-4559-8c2a-0765ba5e8d80)

**Comment by Rist8** [comment]
If you are trying to import a mesh with bones, it's currently not working because of some changes made to bones logic in Release version.

**Comment by Anua831** [comment]
OK, I understand. But this time I only left the mesh, and it still didn't work.

**Comment by Hazuki-san** [comment]
You can create_skinned_appendix and use the original Mita skeleton names.

**Comment by Anua831** [comment]
> 您可以 create_skinned_appendix 并使用原始的 Mita 骨架名称。

Sorry for the late reply, it still doesn't work. In addition, this problem will occur even if I replace the mesh of other templates.

**Comment by PeTcHeNkA** [comment]
![image](https://github.com/user-attachments/assets/823fa691-4692-4c00-a903-a40d31b0cd50)
You need to use the mesh names as circled in the screenshot. 
Also keep in mind that if Armature contains invalid bones that differ from the original bones, this will throw an error.

**Comment by Anua831** [comment]
Yes, it is true that the names in the circled mesh did not match. Now it works properly after modification. I sincerely appreciate everyone's help.
